sync-core/src/main/java/com/cloudant/sync/datastore/DatastoreConstants.java

```diff
@@ -110,4 +110,36 @@
         };
     };
 
+//    Remember the format of all indexes and triggers associated with table X. This information will be needed in step 7 below. One way to do this is to run a query like the following: SELECT type, sql FROM sqlite_master WHERE tbl_name='X'.
```

Comment:
Looks like this comment came from some sort of SQLite guide. Can we reformat it for line length and cite where it comes from? Most importantly we should make clear what this schema upgrade is doing: effectively dropping the unused revid column from localdocs